### PR TITLE
feature(pu): add lightzero mujoco env and related sampled efficientzero configs

### PR DESCRIPTION
**Pull request by puyuan1996**
- Add lightzero mujoco env and related sampled efficientzero configs.

- The benchmark results are as follows::
image

### CODE REVIEW

File: zoo/mujoco/config/mujoco_disc_sampled_efficientzero_config.py [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+            continuous_action_space=continuous_action_space,
+            num_of_sampled_actions=K,
+            model_type='mlp',
+            lstm_hidden_size=256,
+            latent_state_dim=256,
+            self_supervised_learning_loss=True,
+            res_connection_in_dynamics=True,
+            norm_type='BN',
+        ),
+        mcts_ctree=True,
```
Comment (PaParaZz1) [inline zoo/mujoco/config/mujoco_disc_sampled_efficientzero_config.py:72]: simplify config

File: zoo/mujoco/config/mujoco_disc_sampled_efficientzero_config.py [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+    # for halfcheetah, we ignore done signal to predict the Q value of the last step correctly.
+    ignore_done = True
+
+# ==============================================================
+# begin of the most frequently changed config specified by the user
+# ==============================================================
+collector_env_num = 8
+n_episode = 8
+evaluator_env_num = 3
+continuous_action_space = False
```
Comment (PaParaZz1) [inline zoo/mujoco/config/mujoco_disc_sampled_efficientzero_config.py:30]: why False here

File: zoo/mujoco/envs/mujoco_lightzero_env.py [hunk 3]
```diff
@@ -0,0 +1,179 @@
...
+    def observation_space(self) -> gym.spaces.Space:
+        return self._observation_space
+
+    @property
+    def action_space(self) -> gym.spaces.Space:
+        return self._action_space
+
+    @property
+    def reward_space(self) -> gym.spaces.Space:
+        return self._reward_space
```
Comment (PaParaZz1) [inline zoo/mujoco/envs/mujoco_lightzero_env.py:179]: add benchmark figure in readme

**Review by PaParaZz1 — APPROVED** [review]

File: zoo/mujoco/config/mujoco_disc_sampled_efficientzero_config.py (hunk 2 above)
Comment (puyuan1996) [reply zoo/mujoco/config/mujoco_disc_sampled_efficientzero_config.py:30]: 这个key是用在ctree和sez policy里面的，因为是离散化后的动作空间，对于ctree和sez policy而言就相当于离散动作空间，所以是False

File: zoo/mujoco/config/mujoco_disc_sampled_efficientzero_config.py (hunk 1 above)
Comment (puyuan1996) [reply zoo/mujoco/config/mujoco_disc_sampled_efficientzero_config.py:72]: fixed

File: zoo/mujoco/envs/mujoco_lightzero_env.py (hunk 3 above)
Comment (puyuan1996) [reply zoo/mujoco/envs/mujoco_lightzero_env.py:179]: done

**Pull request merged**